src/manifolds/ProbabilitySimplex.jl

```diff
@@ -422,15 +453,93 @@ function riemannian_gradient!(M::ProbabilitySimplex, X, p, Y; kwargs...)
 end
 
 function Base.show(io::IO, ::ProbabilitySimplex{n,boundary}) where {n,boundary}
-    return print(io, "ProbabilitySimplex($(n); boundary=$boundary)")
+    return print(io, "ProbabilitySimplex($(n); boundary=:$boundary)")
 end
 
 @doc raw"""
     zero_vector(M::ProbabilitySimplex, p)
 
-returns the zero tangent vector in the tangent space of the point `p`  from the
+Return the zero tangent vector in the tangent space of the point `p`  from the
 [`ProbabilitySimplex`](@ref) `M`, i.e. its representation by the zero vector in the embedding.
 """
 zero_vector(::ProbabilitySimplex, ::Any)
 
 zero_vector!(::ProbabilitySimplex, X, p) = fill!(X, 0)
+
+@doc raw"""
+    RealProbabilityAmplitudes
+
+An explicit isometric embedding of interior of [`ProbabilitySimplex`] in
+positive quadrant of the [`Sphere`]. Some properties extend to the boundary but not all.
```

Comment:
https://en.wikipedia.org/wiki/Orthant
```suggestion
positive orthant of the [`Sphere`]. Some properties extend to the boundary but not all.
```

Reply:
It would be nice if the docstring explained why it's called this. Because otherwise a name that reflects the two manifolds might be nice.

Reply:
Sure, I will add a comment about it.